lib/collection/src/shard/shard_config.rs

```diff
@@ -8,10 +8,11 @@ use crate::shard::PeerId;
 
 pub const SHARD_CONFIG_FILE: &str = "shard_config.json";
 
-#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
+#[derive(Debug, Deserialize, Serialize, Copy, Clone, PartialEq)]
 pub enum ShardType {
     Local,
     Remote { peer_id: PeerId },
+    Temporary, // same as local, but not ready yet
```

Comment:
I guess it is better as a doc comment
```suggestion
     /// Same as local, but not ready yet
    Temporary,
```